service/internal/env/env.go

```diff
@@ -10,23 +10,24 @@ import (
 const appPrefix = "ARGO_CLOUDOPS"
 
 type Vars struct {
-	AdminSecret    string `split_words:"true" required:"true"`
-	VaultRole      string `envconfig:"VAULT_ROLE" required:"true"`
-	VaultSecret    string `envconfig:"VAULT_SECRET" required:"true"`
-	VaultAddress   string `envconfig:"VAULT_ADDR" required:"true"`
-	ArgoAddress    string `envconfig:"ARGO_ADDR" required:"true"`
-	ArgoNamespace  string `envconfig:"WORKFLOW_EXECUTION_NAMESPACE" default:"argo"`
-	ConfigFilePath string `envconfig:"CONFIG" default:"argo-cloudops.yaml"`
-	SSHPEMFile     string `envconfig:"SSH_PEM_FILE"`
-	GitAuthMethod  string `split_words:"true" required:"true"`
-	GitHTTPSUser   string `envconfig:"GIT_HTTPS_USER"`
-	GitHTTPSPass   string `envconfig:"GIT_HTTPS_PASS"`
-	LogLevel       string `split_words:"true"`
-	Port           int    `default:"8443"`
-	DBHost         string `split_words:"true" required:"true"`
-	DBUser         string `split_words:"true" required:"true"`
-	DBPassword     string `split_words:"true" required:"true"`
-	DBName         string `split_words:"true" required:"true"`
+	AdminSecret    string   `split_words:"true" required:"true"`
+	VaultRole      string   `envconfig:"VAULT_ROLE" required:"true"`
+	VaultSecret    string   `envconfig:"VAULT_SECRET" required:"true"`
+	VaultAddress   string   `envconfig:"VAULT_ADDR" required:"true"`
+	ArgoAddress    string   `envconfig:"ARGO_ADDR" required:"true"`
+	ArgoNamespace  string   `envconfig:"WORKFLOW_EXECUTION_NAMESPACE" default:"argo"`
+	ConfigFilePath string   `envconfig:"CONFIG" default:"argo-cloudops.yaml"`
+	SSHPEMFile     string   `envconfig:"SSH_PEM_FILE"`
+	GitAuthMethod  string   `split_words:"true" required:"true"`
+	GitHTTPSUser   string   `envconfig:"GIT_HTTPS_USER"`
+	GitHTTPSPass   string   `envconfig:"GIT_HTTPS_PASS"`
+	LogLevel       string   `split_words:"true"`
+	Port           int      `default:"8443"`
+	DBHost         string   `split_words:"true" required:"true"`
+	DBUser         string   `split_words:"true" required:"true"`
+	DBPassword     string   `split_words:"true" required:"true"`
+	DBName         string   `split_words:"true" required:"true"`
+	ImageURIs      []string `envconfig:"IMAGE_URIS"`
```

Comment:
Temporary parking space until unified configuration flow is addressed in another PR.